pkg/export/export.go

```diff
@@ -46,10 +46,18 @@ var (
 		Name: "gcm_export_samples_exported_total",
 		Help: "Number of samples exported at scrape time.",
 	})
+	exemplarsExported = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "gcm_export_exemplars_exported_total",
+		Help: "Number of exemplars exported at scrape time.",
+	})
 	samplesDropped = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Name: "gcm_export_samples_dropped_total",
 		Help: "Number of exported samples that were dropped because shard queues were full.",
 	}, []string{"reason"})
+	exemplarsDropped = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "gcm_export_exemplars_dropped_total",
+		Help: "Number of exported exemplars that were dropped because shard queues were full.",
```

Comment:
We should change the help text since it's not just full shard queues causing this, e.g.
```
Number of exported {samples,exemplars} that were intentionally dropped.
```

Reply:
sounds good -- I can change the `samplesDropped` text as well